cms/templates/certificate_page.html

```diff
@@ -96,7 +96,7 @@ <h2>Congratulations, {{ learner_name }}!</h2>
             <div class="certificate-logo">
                 <a href="https://web.mit.edu/" class="mit"></a>
                 <a href="/" class="bc"></a>
-                <img src="{% static 'images/certificates/certificate-bootcamps-logo.png' %}" alt="Bootcamps">
+                <img src="{% static 'images/mit-ol-logo.svg' %}" alt="Bootcamps">
```

Comment:
The MIT Open Learning logo in the [designs](https://www.figma.com/file/GFl0GZREDaWmdRytsNiET2/MIT-Logo-Header-%2F-Footer?node-id=7%3A2675&mode=dev) is bigger than how it appears in the certificate.
<img width="345" alt="image" src="https://github.com/mitodl/bootcamp-ecommerce/assets/34372316/c03a1e73-2d53-48b7-99c4-dc8575500cd6">

Reply:
Good catch. Thanks!